docs/Using-SRIOV-and-Clear-Containers.md

```diff
@@ -0,0 +1,237 @@
+# Setup to use SR-IOV with Clear Containers
+
+Single Root I/O Virtualization (SR-IOV) enables splitting a physical device into
+virtual functions (VF). This enables direct passthrough to virtual machines or
+containers.  For Clear Containers, we have enabled a CNM plugin and necessary
+changes in the runtime to detect virtual functions in a container's 
+network namespace to facilitate making use of SR-IOV for network based devices.
+
+## Install the SR-IOV Docker\* plugin
+
+In order to create a network with VFs associated with it which can be passed to
+Clear Containers, it is necessary to install a SR-IOV Docker plugin. The 
+created network will be based on a physical function (PF) device, and can
+create 'n' containers, where 'n' is the number of VFs associated with the 
+Physical Function (PF).
+
+To install the plugin, follow the [plugin installation instructions](https://github.com/clearcontainers/sriov).
+
+
+## Host setup for SR-IOV
+
+You may need to rebuild your host system's kernel in order to disable 
```

Comment:
Am I right in saying we have the following different ideas/purposes for this section:
1) Introducing that `CONFIG_VFIO_NOIOMMU` needs to be disabled
2) Informing readers that they may need to rebuild the kernel disable `CONFIG_VFIO_NOIOMMU`
3) *Instructing readers on how to disable `CONFIG_VFIO_NOIOMMU` if no rebuild is necessary*
4) Informing readers that adding a PCI may be necessary and introducing the section where the reader can find out if that is necessary.

If so, we need to:
* Add to the beginning of this section: `CONFIG_VFIO_NOIOMMU` must be disabled to setup your host   system for SR-IOV. The method for disabling `CONFIG_VFIO_NOIOMMU` differs depending on the host system's configuration. 
* Separate the sentence saying the host's system kernel needs to be rebuild and a PCI needs to be added. 
* Add instructions on how to disable `CONFIG_VFIO_NOIOMMU` when you do not need to rebuild the kernel.

Reply:
I have changed to wording here.